src/commands/openapi/convert.ts

```diff
@@ -28,7 +28,7 @@ export default class OpenAPIConvertCommand extends BaseCommand<typeof OpenAPICon
   static flags = {
     out: Flags.string({ description: 'Output file path to write converted file to' }),
     title: titleFlag,
-    workingDirectory: workingDirectoryFlag,
+    'working-directory': workingDirectoryFlag,
```

Comment:
Since this rename feels like a breaking change should we support workingDirectory and working-directory but mark the former as deprecated?

Reply:
nevermind just saw the alias!

Reply:
yep you got it! `--workingDirectory` will still work but using it will now emit a deprecation notice